src/pages/profiles/[id]/index.tsx

```diff
@@ -110,6 +111,22 @@ const ProfilePage: NextPage<Props> = ({ errorCode, profile }) => {
             )
             : null
           }
+          {profile.testimonials.length
+            ? (
+              <div className="mt-3 text-left">
+                <h4>Testimonials</h4>
+                <div>
+                  {profile.testimonials.slice(0, 3).map((t, i) => <Testimonial key={i} testimonial={t} />)}
+                </div>
+                {profile.testimonials.length > 1
```

Comment:
We should only show the link to the testimonials page if there are more than 3 testimonials. Otherwise visitors would see the same one to three testimonials in both places

Reply:
This was fixed.